test/jdk/java/net/httpclient/BufferSizePropertyClampTest.java

```diff
@@ -68,13 +69,20 @@ class BufferSizePropertyClampTest {
     private static final List<String> CLIENT_LOGGER_MESSAGES =
             Collections.synchronizedList(new ArrayList<>());
 
+    private static final String EXPECTED_MSG =
+            "ERROR: Property value for jdk.httpclient.bufsize={0} not in [1..16,384]: using default=16,384";
```

Comment:
Doesn't this have an assumption on the formatting of 1 and 16384, and hence, suffers from the same short-coming this fix is aiming to solve? That is, the original `Utils::getIntegerNetProperty` message is as follows:

```
"Property value for {0}={1} not in [{2}..{3}]: using default={4}"
```

Shouldn't we be using this message verbatim instead?